platforms/common/src/CommonKernels.cpp

```diff
@@ -236,15 +236,16 @@ class CommonCalcCustomBondForceKernel::ForceInfo : public ComputeForceInfo {
     }
     void getParticlesInGroup(int index, vector<int>& particles) {
         int particle1, particle2;
-        vector<double> parameters;
+        thread_local static vector<double> parameters;
         force.getBondParameters(index, particle1, particle2, parameters);
         particles.resize(2);
         particles[0] = particle1;
         particles[1] = particle2;
     }
     bool areGroupsIdentical(int group1, int group2) {
         int particle1, particle2;
-        vector<double> parameters1, parameters2;
+        thread_local static vector<double> parameters1;
+        thread_local static vector<double> parameters2;
```

Comment:
You can keep it as one line: `thread_local static vector<double> parameters1, parameters2;`.